bundler/lib/bundler/installer.rb

```diff
@@ -15,7 +15,7 @@ class << self
       Installer.ambiguous_gems = []
     end
 
-    attr_reader :post_install_messages
+    attr_reader :post_install_messages, :fund_metadata
```

Comment:
All the changes in this file can go now, right?

Reply:
Whoops, yes, that was lazy of me. 🙈